README.md

```diff
@@ -52,26 +52,33 @@ where app.json may look something like:
             "sampleInterval": 1000
         }
     },
-    "plugins": {
-        "good": {
-            "opsInterval": 5000,
-            "requestHeaders": true,
-            "reporters": [{
-                "reporter": "good-console",
-                "events": { "response": "*", "ops": "*", "log": "*", "error": "*" }
-            },
-            {
-                "reporter": "good-file",
-                "events": { "response": "*", "error": "*" },
-                "config": "/log/response.log"
-            },
-            {
-                "reporter": "good-file",
-                "events": { "ops": "*" },
-                "config": "/log/ops.log"
-            }]
+    "registrations": [
```

Comment:
instead of registrations requiring an array of objects, you could simplify... if its a string then its the plugin name, if its an object then its the plugin with register and options.

```json
"registrations": [
  {
    "register": "good",
    "options": { }
  },
  "lout"
]
```